README.md

```diff
@@ -0,0 +1,75 @@
+# laravel-weather-apis
```

Comment:
same comment as on the other repo, can we use the generator for this?  Also maybe this header should say something like "Momento Laravel Example"?